### ISSUE TITLE
cant get nicknames to work

### ISSUE THREAD

**Issue by ThatMurloc** [olliz0r/Ledybot#42]
here is a picture of the setup i have currently: https://imgur.com/a/LB9Xw

it will trade to mons with any name instead of just the one i want

**Comment by k20stitch** [comment]
Gonna have to be more specific, nicknames work for me. You name the file what you want the person to nickname (12 characters or less) and then the user trading had to nickname the Pokémon they caught to the same name

**Comment by ThatMurloc** [comment]
> Gonna have to be more specific, nicknames work for me. You name the file what you want the person to nickname (12 characters or less) and then the user trading had to nickname the Pokémon they caught to the same name

here are the steps i took set up ledybot: https://imgur.com/a/Qjyh1

**Comment by k20stitch** [comment]
Okay, now screenshots of the folder containing all the nicknamed Pokémon.

**Comment by ThatMurloc** [comment]
> Okay, now screenshots of the folder containing all the nicknamed Pokémon.

It's on the desktop (see step 3).

**Comment by k20stitch** [comment]
Nope, ledybot should have created a folder and I need to see all the Pokémon files you “nicknamed” in there.

**Comment by k20stitch** [comment]
Also, you can’t delete the specific path, that’s where all the nicknamed monsters are.

**Comment by ThatMurloc** [comment]
I just followed this video here: https://www.youtube.com/watch?v=XrBEZ7qVaro

And Ledybot does create a folder whenever I run it and the folder's name is the name of the pk7 file. Is that supposed to happen?

**Comment by k20stitch** [comment]
if you followed the video then why did you delete the specific path. 

When you drag the pk7 file in, that's the pokemon the user gets if they don't knickname the pokemon they trade. It creates a folder and in that folder you put the nickname specific pokemon in there. So if you want an adamant version, call it adamant.pk7 and the user nicknames their trading pokemon adamant and that's the version he gets. Check out the read me file there's a link to a better description.

In the meantime, this is not an actual issue so it can be closed.

**Comment by ThatMurloc** [comment]
i see, thanks for your help
